### PR TITLE
doc: Add step-by-step manual mod-install example

### PR DESCRIPTION
**Pull request by x10an14**
Add an _OS-agnostic_ **_and_** _Client-agnostic_ (ref: Steam vs Epic Games Launcher) how-to example of how to enable mods.
Motivation for this is to clarify the instructions already existing on the page - especially for those of us unable to leverage SMM in its current state (Linux + Steam).

PS: Added a disclaimer to allay any fears that others could take this as a reason for hounding this community for support in the future.

### CODE REVIEW

**Comment by Panakotta00** [comment]
Please change your request target to the development branch, we don't merge PRs to master directly.
Else I would approve it ^^

**Comment by x10an14** [comment]
> Please change your request target to the development branch, we don't merge PRs to master directly.
> Else I would approve it ^^

Much appreciated @Panakotta00! I assumed you meant the `Dev` branch, which is now the target!

File: modules/ROOT/pages/ManualInstallDirections.adoc [hunk 1]
```diff
@@ -69,3 +69,39 @@ But you can try to name your files so its name comes after the name of the depen
...
+
+== Installing mods on Linux (Steam) as a player - SML version 2.2.1
+
+[WARNING]
+====
+_THIS IS UNSUPPORTED -> NO SUPPORT SHOULD BE EXPECTED FOR ANYONE ATTEMPTING THIS_.
+====
+
+. `cd <game root>/FactoryGame/Binaries/Win64`
+. Navigate to https://github.com/satisfactorymodding/SatisfactoryModBootstrapper/releases/tag/v2.0.11 and download into the `./Win64` folder:
```
Comment (Vilsol) [inline modules/ROOT/pages/ManualInstallDirections.adoc:81]: Please change this to use a `latest` url (https://github.com/satisfactorymodding/SatisfactoryModBootstrapper/releases/latest)

File: modules/ROOT/pages/ManualInstallDirections.adoc [hunk 2]
```diff
@@ -69,3 +69,39 @@ But you can try to name your files so its name comes after the name of the depen
...
+. . `msdia140.dll`
+. . `xinput1_3.dll`
+. `mkdir -p <game root>/loaders && cd <game root>/loaders`
+.  Navigate to https://github.com/satisfactorymodding/SatisfactoryModLoader/releases/tag/2.2.1 and download into the `./loaders` folder:
```
Comment (Vilsol) [inline modules/ROOT/pages/ManualInstallDirections.adoc:85]: Please change this to use a `latest` url (https://github.com/satisfactorymodding/SatisfactoryModLoader/releases/latest)

**Review by Vilsol — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/ManualInstallDirections.adoc (hunk 1 above)
Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc:81]: Is it preferable to ensure backwards compatibility (keep it as is), or add the responsibility to update/correct documentation if anything ever changes?

Would be nice to hear from maintainers (you maybe?) where you'd like to draw that line before I change it =)
Don't feel strongly about it myself, would like to give the choice to the majority of the maintainers.

Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc:81]: @Vilsol ---^

Comment (budak7273) [reply modules/ROOT/pages/ManualInstallDirections.adoc:81]: The docs aren't always updated when new SML updates release, so I would rather have it as a `latest` link. That way we won't have to update it with every release. If there's a problem and a certain old release has to be used, we would ideally put a note about that here.

Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc:81]: Aight, that's two for, and no against in as many days! Change incoming as soon as I get a chance!

Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc]: Done! =)

File: modules/ROOT/pages/ManualInstallDirections.adoc (hunk 2 above)
Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc]: Done! =)

**Comment by x10an14** [comment]
Done @Vilsol =)

File: modules/ROOT/pages/ManualInstallDirections.adoc [hunk 3]
```diff
@@ -69,3 +69,41 @@ But you can try to name your files so its name comes after the name of the depen
...
+.  Navigate to https://github.com/satisfactorymodding/SatisfactoryModLoader/releases/latest and download into the `./loaders` folder:
+. . `SML.pak`
+. . `UE4-SML-Win64-Shipping.dll`
+. `mkdir -p <game root/mods`
+. For each mod you want to install:
+. . First navigate to its page on https://ficsit.app, and go to `Versions` tab.
+. . Then download the version matching your SML version (step 4) into `<game root>/mods`^3^.
+
+1: 22.07.2020 this worked for https://github.com/satisfactorymodding/SatisfactoryModBootstrapper/releases/tag/v2.0.11[this specific version].
+2: 22.07.2020 this worked for https://github.com/satisfactorymodding/SatisfactoryModLoader/releases/tag/2.2.1[this specific version].
```
Comment (Vilsol) [inline modules/ROOT/pages/ManualInstallDirections.adoc:94]: This reference is not referenced anywhere

**Review by Vilsol — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/ManualInstallDirections.adoc (hunk 3 above)
Comment (x10an14) [reply modules/ROOT/pages/ManualInstallDirections.adoc:94]: Thanks for catching that blunder! Fix'd =)

**Review by Vilsol — APPROVED** [review]

**Comment by budak7273** [comment]
Thanks!

**Pull request merged**